README.md

```diff
@@ -40,11 +40,76 @@ npm install --save slackapp
 ```
 
 ## Setup
-//TODO
+You can call the SlackApp function with the following options:
+```js
+const SlackApp = require('slackapp')
+const ConvoStore = require('slackapp-convo-beepboop')
+
+var slackapp = SlackApp({
+  verify_token: process.env.SLACK_VERIFY_TOKEN,
+  convo_store: ConvoStore(),
+  error: (err) => console.log(err),
+
+  // These properties should only be used in testing
+  // if you want to peg it to a specific team's config
+  app_token: '<APP_TOKEN>',
+  app_user_id: '<USER_ID>',
+  bot_token: '<BOT_TOKEN>',
+  bot_user_id: '<BOT_USER_ID>'
+})
+```
 
 ### Token Lookup
-//TODO
-With Beep Boop this is handled out of the box...
+One of the challenges with writing a multi-team Slack app is that you need to make sure you have the appropriate tokens and meta-data for a team when you get a message from them. This lets you make api calls on behalf of that team in response to incoming messages from Slack. You typically collect and store this meta-data during the **Add to Slack** OAuth flow. If you're running on [Beep Boop][beepboop], this data is saved for you automatically. SlackApp has a `token_lookup` option that gives you a convenient hook to load that team-specific meta-data and enrich the message with it.  While you can add whatever meta-data you have about a team in this function, you must set the follwing properties on `req.slackapp.meta` for SlackApp to process requests:
+
++ `app_token` - OAuth `access_token` property
++ `app_user_id` - `user_id` prop from an [auth.test()](https://api.slack.com/methods/auth.test) call using the OAuth `access_token` value
++ `bot_token` - OAuth `bot.bot_access_token` property
```

Comment:
Should add clarification for `bot_token` and `bot_user_id` that these are required _if_ your SlackApp has a bot user